www/includes/easyparliament/templates/html/alert/_alert_controls.php

```diff
@@ -32,16 +32,17 @@
       <span><?= gettext('Edit alert') ?></span>
     </button>
     <?php } else { ?>
-    <input type="hidden" name="mp_step" value="mp_confirm">
```

Comment:
Could we please have some documentation of the flows for future reference, maybe in Standard.php I guess? What is `mp_step` for? I see with this removed there's only one use in a template of `mp_confirm` that's not actually then checked anywhere, is that because it uses the default behaviour? Was this actually creating a new alert each time (that I didn't spot last time)?
(I guess this is a continuation of my 'confused' comment from last review, I'm happy this is fixing all the bugs, but I'm not confident looking back at this in years I will understand what is going on.)